docs/RBAC.md

```diff
@@ -0,0 +1,97 @@
+# Role Based Access Control (RBAC)
+
+### Status: DRAFT
+**This feature is not fully implemented. This document describes the desired imlpementation for Odessey (search v2).**
+
+The search service collects data using a service account with wide cluster access and stores all resources in the database. The API must enforce that results for each user (or service account) only contain resources that they are authorized to access.
+
+## Access to the API
```

Comment:
Should we say that this is `Query API` . As you know, we have another API to post data.